src/battle_util.c

```diff
@@ -9444,22 +9444,22 @@ static inline u32 CalcMoveBasePowerAfterModifiers(struct DamageCalculationData *
         break;
     case ABILITY_PIXILATE:
         if (moveType == TYPE_FAIRY && gBattleStruct->ateBoost[battlerAtk])
-            modifier = uq4_12_multiply(modifier, UQ_4_12(1.2));
+            modifier = GetGenConfig(GEN_CONFIG_ATE_MULTIPLIER) >= GEN_7 ? uq4_12_multiply(modifier, UQ_4_12(1.2)) : uq4_12_multiply(modifier, UQ_4_12(1.3));
```

Comment:
I think this is slightly less verbose?
```suggestion
            modifier = uq4_12_multiply(modifier, UQ_4_12(GetGenConfig(GEN_CONFIG_ATE_MULTIPLIER) >= GEN_7 ? 1.2 : 1.3));
```

Alternatively, maybe a `GetAteAbilityModifier` would be better?